internal/integration-corpus-generator/assets.go

```diff
@@ -0,0 +1,119 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package integration_corpus_generator
```

Comment:
this package may be better named as `genlib`: current implementation fetches the artifacts from the integration corpus generator repo, but once we will update the package-spec to carry those information directly in the package content, it will be more loosing coupled to the integration corpus generator, using just the `Config` and `Fields` parsers and the `Generator`

Reply:
I'd propose to avoid underscores in the name of this package. `corpusgenerator` can be a good name.